src/generic_ui/polymer/instance.html

```diff
@@ -54,8 +54,7 @@
     }
     </style>
 
-    <div id='wrapper' class='{{ instance.isOnline ? "online" : "offline" }}'
-         hidden?='{{model.globalSettings.mode!=ui_constants.Mode.GET}}'>
+    <div id='wrapper' class='{{ instance.isOnline ? "online" : "offline" }}'>
```

Comment:
Does this change mean that the verification UI will also appear in the Share tab?